Sources/PackageModel/UserToolchain.swift

```diff
@@ -744,4 +772,8 @@ public final class UserToolchain: Toolchain {
     public var xctestPath: AbsolutePath? {
         configuration.xctestPath
     }
+
+    public var pluginServerPath: AbsolutePath? {
```

Comment:
what is "plugin server"? is that for toolchain macros?

Reply:
Yep, that's right.

Reply:
probably worth naming it something less generic, maybe swiftCompilerPluginsServerPath if we dont want to mention "macro" specifically

Reply:
`swiftPluginServerPath` maybe, that matches the name of the binary `swift-plugin-server`